spec/warren/app/consumer_add_spec.rb

```diff
@@ -48,7 +48,12 @@
         described_class.invoke(shell,
                                'existing_consumer',
                                { path: path, desc: 'my consumer',
-                                 queue: 'queue_name', bindings: ['topic:exchange_name:c'] })
+                                 # Suggested cop style of %<routing_key_prefix>s but prefer suggesting the simpler
```

Comment:
just wondering whether you need the same comments in the tests?

Reply:
I like to add a comment whenever I disable a cop, just so its clear why I've over-ruled rubocop. Although in this case it may be better to say we're matching the format we suggest to the user.